### DISCUSSION TITLE
[Feature] set default notification settings for all users

### DISCUSSION THREAD

**Discussion by bvarick** [immich-app/immich#13645] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be nice to turn off email notifications for "Album added" and "Album updated" by default, but let users decide to turn them on if they'd like to.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
I've generalized this to #13651
